odbc-api/src/prepared.rs

```diff
@@ -63,6 +63,12 @@ where
         stmt.describe_param(parameter_number).into_result(&stmt)
     }
 
+    /// Returns the number of parameters in an SQL statement.
```

Comment:
Improvement to docstring:

"Number of placeholders which must be provided with [`Self::execute`] in order to execute this statement. This is equivalent to the number of Placeholders used in the SQL string used to create the statement."